charts/internal/cilium/values.yaml

```diff
@@ -268,7 +268,7 @@ global:
 
     # bpf-policy-map-max specifies the maximum number of entries in endpoint
     # policy map (per endpoint)
-    policyMapMax: "16384"
+    policyMapMax: "65000"
 
```

Comment:
I would use a power-of-two value like 65536.
But probably it doesn't matter.

Reply:
+1 for a power of two.